tfhe/Cargo.toml

```diff
@@ -76,7 +76,7 @@ tfhe-versionable = { version = "0.4.0", path = "../utils/tfhe-versionable" }
 wasm-bindgen = { workspace = true, features = [
     "serde-serialize",
 ], optional = true }
-wasm-bindgen-rayon = { version = "=1.2.2", optional = true }
+wasm-bindgen-rayon = { version = "1.3.0", optional = true }
```

Comment:
@mayeul-zama could you rm -rf tfhe/pkg and check the content of the generated pkg for the parallel API ? to know what the packaging will look like